lib/widgets/achievement_widget.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:flutterapp/resources/app_colors.dart';
+import 'package:flutterapp/resources/app_text_sizes.dart';
+import 'package:flutterapp/widgets/progress/arc_progress.dart';
+import 'package:flutterapp/widgets/text/uidisplay_widget.dart';
+import 'package:flutterapp/widgets/text/uitext_widget.dart';
+
+class AchievementWidget extends StatelessWidget {
```

Comment:
Please create a separate package for profile route and put there:
- route
- all widgets that related only to that route and are in separate files, like an achievement

Lets keep only comment widgets in `widgets` package